internal/resources/cluster/resource_cluster.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright © 2021 VMware, Inc. All Rights Reserved.
+Copyright © 2022 VMware, Inc. All Rights Reserved.
```

Comment:
this should not be changed

Reply:
This was not addressed and you marked it resolved ?

cc: @shobha2626

Reply:
@Vasundharashukla please take care in the PR - https://github.com/vmware/terraform-provider-tanzu-mission-control/pull/34